src/bytes_builder.hpp

```diff
@@ -40,6 +40,18 @@ class BytesBuilder {
     }
   }
 
+  void appendDec(uint64_t dec) {
+    for (int i = sizeof(dec) - 1; i >= 0; i--) {
+      bytes.push_back((dec >> (8 * i)) & 0xFF);
+    }
+  }
```

Comment:
```suggestion
template <typename T, typename = is_enable_t<is_convertible_v<T, uint64_t>>>
void appendDec(T dec) {
...
}

```

type_trait 이용하면 밑에 코드랑 중복 줄일 수 있을 것 같은데요